db/migrate/20150710135052_create_package_sub_categories.rb

```diff
@@ -0,0 +1,10 @@
+class CreatePackageSubCategories < ActiveRecord::Migration
+  def change
+    create_table :package_sub_categories do |t|
+      t.integer :package_type_id
+      t.integer :package_category_id
+
+      t.timestamps null: false
+    end
+  end
+end
```

Comment:
@swatijadhav  be good to have indexes on here as well - to avoid whole table scans.